### PR TITLE
LPS-198905 Set CTCollectionId to ThreadLocal before fetching the group

### PR DESCRIPTION
**Pull request by noornajj**
https://liferay.atlassian.net/browse/LPS-198905

### CODE REVIEW

**Comment by noornajj** [comment]
ci:test:sf

File: modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/dto/v1_0/converter/CTEntryDTOConverter.java [hunk 1]
```diff
@@ -54,12 +56,41 @@ public CTEntry toDTO(
...
+				return _language.get(locale, "deleted");
+			}
+		}
+
+		if (ctEntry.getChangeType() == CTConstants.CT_CHANGE_TYPE_ADDITION) {
+			return _language.get(locale, "added");
+		}
+
+		return _language.get(locale, "modified");
+	}
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/dto/v1_0/converter/CTEntryDTOConverter.java:78]: @noornajj I believe you accidentally added this change back when you resolved the conflicts. It was removed in another change I sent yesterday.

**Comment by noornajj** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:forward

**Comment by pei-jung** [comment]
ci:forward:force

**Pull request closed without merging**